INSTALL.md

```diff
@@ -12,25 +12,25 @@ To install the package:
 
 ```bash
 cd <checkout-dir>
-python setup_sklearnex.py install --single-version-externally-managed --record=record.txt
+python setup_sklearnex.py install
 ```
 
 To install the package in the development mode:
 
 ```bash
 cd <checkout-dir>
-python setup.py develop --no-deps
+python setup_sklearnex.py develop
 ```
 
 To install scikit-learn-intelex without the dependency on daal4py:
 
 ```bash
 cd <checkout-dir>
-python setup_sklearnex.py install
+python setup_sklearnex.py install --single-version-externally-managed --record=record.txt
 ```
 ```bash
 cd <checkout-dir>
-python setup.py develop
+python setup_sklearnex.py develop --no-deps
 ```
 
 ⚠️ Keys `--single-version-externally-managed` and `--no-deps` are required so that daal4py is not downloaded after installation of Intel(R) Extension for Scikit-learn
```

Comment:
As we are using keys here it might be worth to explain what each of them would do. 
I can get from --no-deps option is that it will force installer to use oneDAL/daal4py that we have and not one that would be downloaded. But i can't understand what single-version switch will do.